Assets/Scripts/Abstract Classes/PlayerClass.cs

```diff
@@ -32,6 +37,14 @@ public override void Start()
      */
     abstract protected void DrawCard();
 
+    public abstract void UnRenderHand();
+
+    public void PleaseRenderMyHand()
```

Comment:
It's a bit strange that `UnRenderHand` is declared as public, but `RenderHand` is declared as protected. Why, if we're just going to write a trivial forwarding function like this?

Let's remove this, and simply declare both `UnRenderHand` and `RenderHand` as public.

Reply:
agree with the sentiment but I wished to avoid merge conflicts